src/client_side.cc

```diff
@@ -1351,13 +1351,13 @@ ConnStateData::parseHttpRequest(const Http1::RequestParserPointer &hp)
     ClientHttpRequest *http = new ClientHttpRequest(this);
 
     http->req_sz = hp->messageHeaderSize();
-    Http::Stream *result = new Http::Stream(clientConnection, http);
+    auto result = new Http::Stream(clientConnection, http);
```

Comment:
seems to be an unnecessary change.

Reply:
As I'm touching code near here, I applied AAA. I can revert if it's a problem